service/pokemon_test.go

```diff
@@ -0,0 +1,61 @@
+package service_test
```

Comment:
We will take a look on how to do testing on a further workshop, just as a note, VSCode can generate testing templates for a whole file or a single function, which is how we usually do testing on real life projects, those templates are very useful.

When you're on a file or inside a function you can do CMD+Shift+P and start typing `Generate test` or something like that, then a couple of options will appear that will let vscode automatically generate testing templates.

Reply:
Thank you